Russian UI localisation update

PF4Public opened the pull request:
Translated new strings, but there are issues:
* Zanata seems to spoil plurals. You should do something about it. See here: https://github.com/FontManager/font-manager/commit/fc09a43c475ca03d52b6f451c62c96c22a00e334#diff-92817e54c697fd6395db7e9eca8f2468538e83db228823136eaa96d834d74f76L34-L38
* Unfortunately not all of the new strings are applied:
![image](https://user-images.githubusercontent.com/10319700/100526068-b0913200-31d6-11eb-8836-f981a614f396.png) I suppose, that strings need to be updated to correspond to source changes and then we need to review translations once again. I did a quick googling, but ufortunately couldn't find any information on how it should be done with meson, so could not perform it myself.

This probably also applies to my recent German translations. And also highly likely to recent Dutch translation.

How do we proceed?

JerryCasiano commented:
Can you check the latest commit and see if that fixed the issue with loading certain strings. I think I got all of them but maybe not.

The plural forms issue I'll have to look into and see why it's doing that and how to stop it.

PF4Public commented:
Ah, so the issue was not what I thought it was.

It got better, but now it is clear, that there are some strings, which are even absent from po files:
![image](https://user-images.githubusercontent.com/10319700/100528006-fa831380-31e8-11eb-9ae7-77082c3ff97e.png)
Unknown Vendor
![image](https://user-images.githubusercontent.com/10319700/100528049-66657c00-31e9-11eb-91aa-1bb445f635e2.png)
Download Family
![image](https://user-images.githubusercontent.com/10319700/100528071-aa588100-31e9-11eb-8c30-0fc1141c4a98.png)
Should suggestions be translated? They look partly translated, but I admit, this could be a problem, if they are obtained from Google via API.
![image](https://user-images.githubusercontent.com/10319700/100528097-fe636580-31e9-11eb-8d83-d3ef381ccb15.png)
And also the left panel. Drop-down menu under "Sort Order" is also not translated.

PF4Public commented:
I've updated all the strings from pot file and added real plurals this time, but they do not work:
![image](https://user-images.githubusercontent.com/10319700/100543296-59d03a80-3260-11eb-8849-11fdf2a5821d.png)

I suppose you shouldn't be trying to translate strings within `ngettext`:
https://github.com/FontManager/font-manager/blob/4d0ce12ed83a7857f77b75975a380d5b958c9fb2/src/font-manager/CellRenderer.vala#L90-L92
https://github.com/FontManager/font-manager/blob/4d0ce12ed83a7857f77b75975a380d5b958c9fb2/src/font-manager/web/google/PreviewPane.vala#L276-L278

Removing `_()` got me proper plurals:
![image](https://user-images.githubusercontent.com/10319700/100543631-41611f80-3262-11eb-88fd-4137668cb83b.png)

JerryCasiano commented:
I think the changes in master are enough to display all the strings in your translation except for those which are marked fuzzy, which from what I understand is normal.

With the last commit, once I removed all `# , fuzzy` entries all the translated strings were visible.

Those ngettext calls should be ok, I think.

Can you please confirm this when you get a chance?

Looking into switching away from Zanata to Weblate at this point, Zanata is on life support anyway and it's kind of a pain to use.

JerryCasiano commented:
I'm merging what you have at this point because Weblate has already started submitting pull requests.

Feel free to open an issue or another pull request for anything that still needs to be resolved before release.

Thanks again.
